crmsh/ui_cluster.py

```diff
@@ -231,24 +232,29 @@ def looks_like_hostnames(lst):
                                    help="Configure corosync with second heartbeat line")
         network_group.add_argument("-I", "--ipv6", action="store_true", dest="ipv6",
                                    help="Configure corosync use IPv6")
-        network_group.add_argument("--qnetd-hostname",
+
+        qdevice_group = parser.add_argument_group("QDevice configuration", "Options for configuring QDevice and QNetd.")
+        qdevice_group.add_argument("--qnetd-hostname",
                                    dest="qdevice", metavar="HOST",
                                    help="HOST or IP of the QNetd server to be used")
-        network_group.add_argument("--qdevice-port",
+        qdevice_group.add_argument("--qdevice-port",
                                    dest="qdevice_port", metavar="PORT", type=int, default=5403,
                                    help="TCP PORT of QNetd server(default:5403)")
-        network_group.add_argument("--qdevice-algo",
+        qdevice_group.add_argument("--qdevice-algo",
                                    dest="qdevice_algo", metavar="ALGORITHM", default="ffsplit",
                                    help="QNetd decision ALGORITHM(ffsplit/lms, default:ffsplit)")
```

Comment:
I have checked a bit the `argparser` library, and I have found the `choice` option;
https://docs.python.org/3/library/argparse.html#choices
I think we could use it here, and all of the other options that have a limit set of options. This would help us to avoid not valid option, what do you think?

Reply:
OK, change it, that saves lots of testing codes:)